src/main/resources/sql/select2-1-select2-5.sql

```diff
@@ -0,0 +1,40 @@
+-- а) Найти имена покупателей, возраст которых от 16 до 35.
+SELECT
+  name,
+  age
+FROM customers
+WHERE age >= 16 AND age <= 35;
+
+-- б) Найти имена покупателей, фамилия которых оканчивается на 'OV'
+SELECT
+  name,
+  surname
+FROM customers
+WHERE surname ILIKE '%OV';
+
+-- в) Найти название самого дорогого по закупке товара, в имени которого присутствует буква 'V',
+--    но не первая и не последняя.
+SELECT
```

Comment:
Please execute query with this values

INSERT INTO
  products (name, purchase_price)
VALUES
  ('vproduct_name', 32.2)
  , ('productv_name', 92.2)
  , ('productv_name', 92.2)
  , ('vproductv_name', 92.2)
  , ('prodVuct_name', 22.2)
  , ('product_namev', 103.2)
  , ('vprovduct_namev', 32.2)
  , ('VVproduct_name', 42.2)
  , ('vproduct_name', 92.2)
  , ('product_namev', 92.2)
  , ('product_name', 92.2)
  , ('prodVVuct_name', 2.2);

Reply:
EXTERNAL query is wrong. Some values, that not satisfy condition can have such price

Reply:
Yep, thank you

Reply:
fixed